fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/Repository.java

```diff
@@ -23,19 +23,4 @@
  */
 public interface Repository {
```

Comment:
Create a JIRA for removing Repository interface and all dependencies.

Reply:
https://jira.duraspace.org/browse/FCREPO-3089